src/schema/query/activityLogs.query.ts

```diff
@@ -53,22 +53,36 @@ type ActivityLogsArgs = {
 };
 
 /**
- * GraphQL query to list all activitiesLogs.
+ * GraphQL query to list all activity logs.
  */
 export default {
   type: ActivityLogConnectionType,
   args: {
     first: { type: GraphQLInt },
     afterCursor: { type: GraphQLID },
     filter: { type: GraphQLJSON },
+    userId: { type: GraphQLString },
+    applicationId: { type: GraphQLString },
   },
-  async resolve(parent, args: ActivityLogsArgs) {
-    // Make sure that the page size is not too important
+  async resolve(
+    parent,
+    args: ActivityLogsArgs & { userId?: string; applicationId?: string }
```

Comment:
@TaiKamilla 
the ActivityLogsArgs type is supposed to be extended in order to work with the userId & applicationId
Adding them like how you did is incorrect